mlir/include/mlir/Dialect/Utils/StaticValueUtils.h

```diff
@@ -109,9 +109,8 @@ SmallVector<OpFoldResult> getMixedValues(ArrayRef<int64_t> staticValues,
 /// Decompose a vector of mixed static or dynamic values into the
 /// corresponding pair of arrays. This is the inverse function of
 /// `getMixedValues`.
-std::pair<ArrayAttr, SmallVector<Value>>
-decomposeMixedValues(Builder &b,
-                     const SmallVectorImpl<OpFoldResult> &mixedValues);
+std::pair<SmallVector<int64_t>, SmallVector<Value>>
```

Comment:
I am not sure this is needed.

Reply:
Yeah, it seems this patch does not necessarily require this method. If the helper build method seems fine(mentioned in the next comment), I will address this comment together with the next one.